py/selenium/webdriver/blackberry/webdriver.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python
+#
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+import platform
+import time
+import subprocess
+
+from selenium.webdriver.remote.command import Command
+from selenium.webdriver.remote.webdriver import WebDriver as RemoteWebDriver
+from selenium.common.exceptions import WebDriverException
+
+class WebDriver(RemoteWebDriver):
+    """
+    Controls the BlackBerry Browser and allows you to drive it.
+
+    """
+    def __init__(self, device_pass, bb_tools_dir, hostip=None, port=1338,desired_capabilities={}):
+        """
+        Creates a new instance of the BlackBerry driver.
```

Comment:
can you add some description of each of the parameters?

Can you change device_pass to be device_password

bb_tools_dir, is there any default? or any discoverable way to find it? or a normal environment variable recommended to be set for developers when they use blackberry tools?